gradle/publish.gradle

```diff
@@ -1,14 +1,22 @@
+ext {
+    GROUP_ID = 'com.novoda'
```

Comment:
these are all variables, not constants, so we can avoid to use the ALL_CAPS format here